docs/network-flow-visibility.md

```diff
@@ -85,14 +84,15 @@ Visualization Flows Dashboard">
 Visualization Flow Dashboard"> 
 
 ### Flow Records 
-Flow Records dashboard shows raw flow records over time with filter.
+Flow Records dashboard shows the raw flow records over time with support 
+for filters.
 
 <img src="https://s3-us-west-2.amazonaws.com/downloads.antrea.io/static/flow-visualization-flow-record.png" width="900" alt="Flow
 Visualization Flow Record Dashboard">
 
 ### Node Throughput
-Node Throughput dashboard visualizes inter-node and intra-node traffic
-by aggregating pod traffic per node.
+Node Throughput dashboard shows the visualization of inter-Node and 
```

Comment:
IMO, here with inter-Node, intra-Node and per Node, we have to use smaller case because we are not referencing a particular node. Similarly for "all the Pod traffic",  we need to use smaller because we are characterizing the traffic as Pod traffic and not referencing a particular pod.

Reply:
Sounds great. How about changing it to:
Node Throughput dashboard provides an insight in inter-Node and intra-Node traffic through aggregated Pod traffic.